file-formats/csnm/example/z_aff_example_csnm.csnm.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
@larshp I am not sure. Do we need to enable object type CSNM in abaplint? If so, I will create an issue in the abaplint project as suggested by the error message.

I am just wondering because this is not the first new object type in our repository.

Reply:
fix the folder name to follow the convention and all should be good

Reply:
@michaelbelenki Would you move the example files to the folder "example**s**", please?